src/xmipp/libraries/data/grids.h

```diff
@@ -490,6 +495,11 @@ class Grid
     /* Structure --------------------------------------------------------------- */
     std::vector<SimpleGrid>   LG;                 // List of grids
 public:
+    /** Destructor.
+     * Explicitly defined to avoid SonarCloud complaining
+     */
+    ~Grid() {}
```

Comment:
The same problem here:
Instead of adding (basically default) destructor, remove the copy assignment operator and follow the rule of Zero.

Reply:
Done